src/api/routers/requests.py

```diff
@@ -0,0 +1,38 @@
+from fastapi import APIRouter, Depends
+
+from src.api.request_models.request import RequestList
+from src.api.response_models.request import RequestDB
+from src.core.services.request_service import RequestService, get_request_service
+
+router = APIRouter()
+
+
+@router.post(
+    '/',
+    response_model=list[RequestDB],
+    response_model_exclude_none=True,
+    summary=("Получить информацию обо всех заявках смены"
+             "с возможностью фильтрации"),
+    response_description="Полная информация обо заявках смены.",
+)
+async def get_list_all_requests_on_project(
+    request_list: RequestList,
+    request_service: RequestService = Depends(get_request_service),
```

Comment:
Не нужно использовать дополнительный метод, сделай как я привожу пример ниже. А метод удали.
```suggestion
    request_service: RequestService = Depends(),
```

Reply:
Без этого метода не работает 
raise fastapi.exceptions.FastAPIError(
fastapi.exceptions.FastAPIError: Invalid args for response field! Hint: check that <class 'sqlalchemy.ext.asyncio.session.AsyncSession'> is a valid pydantic field type

Reply:
Смержи к себе последнюю ветку девелоп и попробуй ещё раз.